simple/imm_data.c

```diff
@@ -105,20 +121,20 @@ static int alloc_ep_res(struct fi_info *fi)
 	cq_attr.size = rx_depth;
 	ret = fi_cq_open(dom, &cq_attr, &rcq, NULL);
 	if (ret) {
-		printf("fi_cq_open() send comp %s\n", fi_strerror(-ret));
+		FI_PRINTERR("fi_cq_open: recv completion", ret);
```

Comment:
This would probably not look good when printed since FI_PRINTERR put the parentheses "()" after the string. May be we need to modify FI_PRINTERR to accept additional string or variadic arguments?